README.md

```diff
@@ -1,5 +1,11 @@
-Zulip
-=====
+**[Zulip -- getting started](#zulip-getting-started)** |
+**[Installing for dev](#installing-the-zulip-development-environment)** |
+**[Installing for production](#running-zulip-in-production)** |
+**[Ways to contribute](#ways-to-contribute)** |
+**[How to get involved](#how-to-get-involved-with-contributing-to-zulip)** |
+**[License](#license)**
+
+# Zulip -- getting started
```

Comment:
Maybe we should make the first heading just "Zulip"?  Is shorter and probably clearer